<launch>
  <!-- DEPRECATED!!!!!!  This is a temp testing launcher -->  
    
   < wont run!
   
   
  <!-- Launch the rfid grid map node -->
  <node pkg="rfid_grid_map" type="rfid_grid_map_node" name="rfid_grid_map_node" output="screen">
      <remap from="/rfid_grid_map/lastTag" to="/lastTag"/>	
      
  </node>
  <!-- Launch the grid map visualizer -->
  <node pkg="grid_map_visualization" type="grid_map_visualization" name="grid_map_visualization" output="screen">
    <rosparam command="load" file="$(find rfid_grid_map)/config/visualization.yaml" />
  </node>
  
  <!-- Launch RViz with the demo configuration -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find rfid_grid_map)/rviz/rfid_gridmap.rviz" />
</launch>
